### PR TITLE
faq: add-sdwire-faq-section

### PR DESCRIPTION
**Pull request by Stanislaw-bnk**

### CODE REVIEW

File: docs/transparent-validation/sd-wire/faq.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# SDWire Frequently asked questions (FAQ)
+
+## Will the SDWire be compatible with the Odroid-XU4?
```
Comment (pietrushnic) [inline docs/transparent-validation/sd-wire/faq.md:3]: @Stanislaw-bnk, this is incorrectly formulated. You use reverse logic. The question should be something like "How do you verify the compatibility of SDWire?" ort something like that.

File: docs/transparent-validation/sd-wire/faq.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+For usage documentation, please visit SDWire [usage validation](https://docs.dasharo.com/transparent-validation/sd-wire/usage-validation/) 
+section.
+
+Please note that the "First use" has already been completed during the pre-sale
+validation of the SDWire. However, we haven't conducted compatibility tests on
+the Odroid-XU4, so we cannot provide specific details regarding compatibility
+with those devices. You can learn more about the RTE platform
+[here](https://shop.3mdeb.com/shop/open-source-hardware/rte/).
+
+## Can I use this product to remotely download photos from a DSLR camera? I'm concerned about speed.
```
Comment (pietrushnic) [inline docs/transparent-validation/sd-wire/faq.md:17]:
```suggestion
## Can I use SDWire to download photos from a DSLR camera remotely?
```

File: docs/transparent-validation/sd-wire/faq.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+* [Tizen Wiki](https://wiki.tizen.org/SDWire)
+
+These resources will provide you with comprehensive guidance on how to make the
+most of the product for your specific needs.
+
+## What are the speed limitations, specifically the maximum speed for reading and writing to an SD card?
+
+We conducted a simple test involving multiple attempts to flash an SD card
+inserted into SD Wire. The data transfer rates for these successive attempts
+ranged between 12 MB/s and 21 MB/s.
```
Comment (pietrushnic) [inline docs/transparent-validation/sd-wire/faq.md:35]: We should be clear about what hardware we used to test. We can link to the compatibility section.

**Review by pietrushnic — CHANGES_REQUESTED** [review]
We should also have a place where we can gather feature requests. @macpijan what do you think is dasharo-issues or OSFV, or maybe https://github.com/3mdeb/SDWire good place to gather feature requests?

@rafkoch we need roadmap focused approach to evolution of this product.

File: docs/transparent-validation/sd-wire/faq.md (hunk 1 above)
Comment (Stanislaw-bnk) [reply docs/transparent-validation/sd-wire/faq.md]: fixed in the latest commit

File: docs/transparent-validation/sd-wire/faq.md (hunk 3 above)
Comment (Stanislaw-bnk) [reply docs/transparent-validation/sd-wire/faq.md]: fixed in the newest commit

**Comment by Stanislaw-bnk** [comment]
The pre-commit for the latest fix show errors. 
The reason is too long heading. How can I easily break/wrap the heading?
Screenshot from 2023-11-23 10-40-22

**Comment by pietrushnic** [comment]
@Stanislaw-bnk The fact you have so long time is a problem in itself, you should shorten that question. Last resort you can disable checks per line. Example: https://github.com/Dasharo/docs/blob/master/docs/osf-trivia-list/dasharo.md?plain=1#L339

**Comment by Stanislaw-bnk** [comment]
fixed in https://github.com/Dasharo/docs/pull/712/commits/962058dde6a868eec4e40940c6a0a20d9aefc064

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**